causal_testing/json_front/json_class.py

```diff
@@ -307,20 +307,29 @@ def _setup_test(self, causal_test_case: CausalTestCase, test: Mapping) -> Estima
         :returns:
                 - estimation_model - Estimator instance for the test being run
         """
-        minimal_adjustment_set = self.causal_specification.causal_dag.identification(causal_test_case.base_test_case)
-        treatment_var = causal_test_case.treatment_variable
-        minimal_adjustment_set = minimal_adjustment_set - {treatment_var}
-        estimator_kwargs = {
-            "treatment": treatment_var.name,
-            "treatment_value": causal_test_case.treatment_value,
-            "control_value": causal_test_case.control_value,
-            "adjustment_set": minimal_adjustment_set,
-            "outcome": causal_test_case.outcome_variable.name,
-            "effect_modifiers": causal_test_case.effect_modifier_configuration,
-            "alpha": test["alpha"] if "alpha" in test else 0.05,
-        }
+        estimator_kwargs = {}
         if "formula" in test:
+            if test["estimator"] != (LinearRegressionEstimator or LogisticRegressionEstimator):
+                raise TypeError(
+                    "Currently only LinearRegressionEstimator and LogisticRegressionEstimator supports the use of "
+                    "formulas"
+                )
             estimator_kwargs["formula"] = test["formula"]
+            estimator_kwargs["adjustment_set"] = {}
```

Comment:
Please can this be `None` instead of empty? A minor thing, but it would help to distinguish the causal semantics of "we don't need an adjustment set because there is no confounding" and "we don't provide an adjustment set because we provide a formula instead"